ChangeLog.d/cmake_add_subdirectory_support.txt

```diff
@@ -0,0 +1,4 @@
+Changes
+   * Add aliases for libraries so that the normal MbedTLS::* targets 
```

Comment:
minor nit, this line has a stray space at the end of it.

Reply:
sorry about that - fixed